2020-hold-my-context.md

```diff
@@ -0,0 +1,58 @@
+# Hold my context!
+
+☝️  _the presentation title_☝️
+
+## Short summary for display
+
+Preserving contextual data across asynchronous calls have been always tricky
+in Node.js land. For example, how to pass information about the current user
+or a correlation (transaction) id from the code processing incoming HTTP
+requests to the code calling a backend web service? Traditional runtimes like
+C and Java can utilize thread-local storage, which does not work for Node's
+single-threaded event loop.
+
+The talk will describe different solutions people have been using (and
+abandoning) over the past 6+ years, dive into history of domains and async
+hooks, explain what's so difficult about context keeping, and finally offer
+bright future that's coming with the recently introduced `AsyncLocalStorage` API.
```

Comment:
I don't know if Vladimir thinks the future is bright! https://github.com/nodejs/TSC/issues/807

Reply:
Good point. My impression was that https://github.com/nodejs/node/pull/26540 is on a good way to get landed soon. I'll mention in the abstract that the final message of my talk my change depending on the outcome of current discussions in Node.js core.